app/src/main/res/menu/menu_reading_options.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto">
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:id="@+id/options_menu"
+  >
```

Comment:
This should go in single line `android:id="@+id/options_menu">`

Reply:
done